client/src/state/actions/saveAction.js

```diff
@@ -0,0 +1,16 @@
+//Saving what user types in input as payload. Will come back to this later
+// export function saveExercise(exerciseData){
+//     return {
+//         type: 'SAVE_EXERCISE',
+//         payload: exerciseData.target.value,
+//         key: exerciseData.target.name
+//     }
+// }
+
+export function saveExercise(exerciseData){
```

Comment:
nit - if this is going to be the only action in this file, the filename should match the action name. Unless you were thinking 'saveAction.js' would include other save-related actions, in which case that could make sense.